README.md

```diff
@@ -1,86 +1,113 @@
 ![Banner](Documentation/Images/Banner.png)
-## Co-op multiplayer RPG and utilities built with Unity Netcode for GameObjects
+<br><br>
+
+# Boss Room: a Co-op, Multiplayer RPG Sample
+###  Made with and Including Utilities for Netcode for GameObjects
+<br>
+
+[![UnityVersion](https://img.shields.io/badge/Unity%20Version:-2021.3%20LTS-57b9d3.svg?logo=unity&color=2196F3)](https://unity.com/releases/editor/qa/lts-releases#:~:text=February%2014%2C%202023-,LTS%20Release,2021.3.18f1,-Released%3A%20February)
```

Comment:
Boss Room is still on 2021.3.15, so the readme should point to that version for the time being.